.github/workflows/ci.yml

```diff
@@ -85,6 +85,38 @@ jobs:
         chmod +x codecov
         ./codecov
 
+  #
+  # Android build job
+  #
+  android:
+    runs-on: macos-latest
+    if: "! contains(toJSON(github.event.commits.*.message), '[skip ci]')"
+
+    # Definition of the build matrix
+    strategy:
+      matrix:
+        java: [ 11 ]
+        android-api: [26, 27, 28, 29, 30, 31]
```

Comment:
I don't think we want to test so many Android releases, since we don't have a lot of CPU capacity on GitHub actions. Instead, I think only having 26 (which is the oldest version that we still support) is sufficient. If we notice gaps in Android support, we can always extend the matrix, but for now I think just 1 Android version is good enough.

Reply:
Good point, very likely if it works on the oldest, it'll work elsewhere too.

Reply:
That said, I would follow this up with a PR for running 33, because that has full on Java 11 support, which can mess things up.

Reply:
Kept only `26`